plugin/storage/cassandra/Dockerfile

```diff
@@ -1,5 +1,4 @@
-# TODO: replace this by the final Cassandra image
-FROM jpkroehling/cassandra
+FROM cassandra:3.11
```

Comment:
Don't have a strong opinion either way. I think generally later versions of cqlsh work fine with older versions of Cassandra, but not vice versa.